ElementX/Sources/Screens/HomeScreen/View/HomeScreen.swift

```diff
@@ -76,6 +76,17 @@ struct HomeScreen: View {
                 userMenuButton
             }
         }
+        .onChange(of: visibleItemIdentifiers, perform: { _ in
+            if !hasTriggeredInitialVisibleItemUpdate {
+                updateVisibleRange()
+                hasTriggeredInitialVisibleItemUpdate = true
+            }
+        })
```

Comment:
I'm sure something will be grumbling about this somewhere, but:
```suggestion
        .onChange(of: visibleItemIdentifiers) { _ in
            if !hasTriggeredInitialVisibleItemUpdate {
                updateVisibleRange()
                hasTriggeredInitialVisibleItemUpdate = true
            }
        }
```